apps/settings/main_controller_prompt_none.cpp

```diff
@@ -10,7 +10,7 @@ constexpr SettingsMessageTree s_modelMenu[] =
   {SettingsMessageTree(I18n::Message::MathOptions, s_modelMathOptionsChildren),
     SettingsMessageTree(I18n::Message::Brightness),
     SettingsMessageTree(I18n::Message::DateTime, s_modelDateTimeChildren),
-    SettingsMessageTree(I18n::Message::Language),
+    //SettingsMessageTree(I18n::Message::Language),
```

Comment:
Nice try, but you should check for device model as the language menu is needed on N0110.